docs/fieldnorm/life_cycle.md

```diff
@@ -1,7 +1,56 @@
\ No newline at end of file
+2.      Requirements
+This process involves gathering, documenting, and analysing the software requirements. It includes capturing both functional and non-functional requirements that the software must fulfil to meet the intended use and comply with regulatory standards. One strategy for capturing this, which is also well suited for genetic laboratories is to use user stories, as stakeholders within a genetic laboratory have very specific needs but are often not able to translate those in well defined technical requirements.
+
+Guidelines should support the following processes to gather, convert, track and update the requirements during a project:  
+
+  a.       Collaborate closely with end users of the software when defining requirements.
+
+  b.       Convert end users' requirements into small individual development tasks.
+
+  c.       Utilize a task management system integrated with your Git development system to manage tasks.
+
+  d.       Maintain communication with end users throughout the development life cycle in order to prevent unexpected adjustments to requirements in later stages of development.
+
+A widely adopted software development strategy involves dividing a development project over several periods. It is advised to use an approach that enables to release partial deliverables to users each period, which enables user feedback upon each deliverable during the entire lifetime of the project, and provides ample opportunities for clarification of requirements or adapt their priorities.   
+
+3.      Design
+Here the software architecture is defined, describing the high-level structure, components, interfaces, and interactions of the software system. The architectural design lays the groundwork for the detailed design and implementation phases.
+
+4.      Development
+The development stage will be the big bulk of a project, here code is created that will eventually define the software.
+
+  a.       Usage of a decentralized code management system, such as git, together with a centralised repository, such as Github, Gitlab or Bitbucket, that is seamlessly integrated with a task management system, is strongly recommended.
+
+  b.       Have a coding policy that covers which language is used, how the language is formatted, do’s and don’ts, the use of inline documentation. This policy is not to be strictly enforced, but will open discussions on how the team likes its code and will lead to a more concise and unified coding strategy.
+
+  c.       Write code with unit and integration tests. Moreover, conducting regular code reviews is highly recommended. These steps are crucial for code verification and will be invaluable during software maintenance, testing, and release phases. It enables a wider understanding of parts of the code as unit tests often embody general usage as well as very specific exceptions and code reviews enforce colleagues to have a look and understand what your code is doing.
```

Comment:
"write code with unit and integration tests" <- we can't cover everything with unit tests. I think we should use "testing" and refer to the section verification for details

Reply:
I have removed the "crucial for code verification" part and made all Development points optional